### PR TITLE
Update the place bet handler to take an EconConfigLoader

### PR DESCRIPTION
**Pull request by ajlacey**
There is now a new setup.EconConfigLoader package which can be used to load the values from setup.yaml across the entire application. This MR implements setup.EconConfigLoader in a few functions.

### CODE REVIEW

**Comment by pwdel** [comment]
* We likely need a placebethandler test function since this is such a critical function however that seems like its own ticket.
* For now I tested with the following:

```
0. Game setup had initialBetFee of 1 and all other fees 0.
1. Starting with an account balance of -118, traded on market for first time in the amount 1 in the YES direction.
2. Resultant balance was -120 as expected.
3. Traded again, +1 in YES direction, resultant balance was -121 as expected.
4. 2 YES shared owned.
5. Both bets shown

**Review by pwdel — COMMENTED** [review]
Tested and showing comments.

We probably need at least a simpe placebethandler test since it's such an important function. We could mimic the manual test that I did for now. However this could be another ticket.

Concentrating on this one MR, I would just question the need to create an error cache for POST methods if that's already handled by the server, which I'm not sure if it is handled quite like what you are looking for.

File: backend/handlers/bets/placebethandler.go [hunk 1]
```diff
@@ -13,91 +12,82 @@ import (
...
-	// Validate the final bet before putting into database
-	if err := betutils.ValidateBuy(db, &bet); err != nil {
-		http.Error(w, err.Error(), http.StatusBadRequest)
-		return
+func PlaceBetHandler(loadEconConfig setup.EconConfigLoader) func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// Only allow POST requests
+		if r.Method != http.MethodPost {
+			http.Error(w, "Method not supported", http.StatusMethodNotAllowed)
+			return
```
[the remark this answers is not in the document]
Comment (ajlacey) [reply backend/handlers/bets/placebethandler.go:20]: This is a carryover from the original handler function. I'm not actually sure how the router handles invalid methods. I think if the router does handle that, we can update all of the handlers in a different pr.

[the remark this answers is not in the document]
Comment (j4qfrost) [reply backend/handlers/bets/placebethandler.go:20]: @pwdel is correct, here is the error that bubbles up. https://github.com/gorilla/mux/blob/db9d1d0073d27a0a2d9a8c1bc52aa0af4374d265/route.go#L58

**Review by j4qfrost — APPROVED** [review]
I think for now we should just merge the change. We should add the refactor for the .Methods("POST") check in another PR. Looks fine aside from that.

File: backend/handlers/bets/placebethandler.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (ajlacey) [reply backend/handlers/bets/placebethandler.go:20]: Nice to see that that's handled automatically for us. Here's a ticket to the refactor for removing this check on each handler. #330

**Pull request merged**